web/src/compose_banner.ts

```diff
@@ -50,6 +50,8 @@ export const CLASSNAMES = {
     zephyr_not_running: "zephyr_not_running",
     generic_compose_error: "generic_compose_error",
     user_not_subscribed: "user_not_subscribed",
+    // uploads
+    upload_banner: "upload_banner",
```

Comment:
You probably don't want to add this here -- it will give the upload banner another close click handler, but that's handled specially in upload.js

(see code in `compose.js`:

```js
    for (const classname of Object.values(compose_banner.CLASSNAMES)) {
        const classname_selector = `.${CSS.escape(classname)}`;
        $("body").on("click", `${classname_selector} .main-view-banner-close-button`, (event) => {
            event.preventDefault();
            $(event.target).parents(classname_selector).remove();
        });
    }
```
)

Reply:
The easiest way I can think of to solve this is by directly including the query to remove uploads, like this: `$(`#compose_banners .upload_banner`).remove();`. How do you feel about this approach?

Reply:
Made an update to the pull request by removing `upload_banner` from the `CLASSNAMES`.

Reply:
Yeah `$('#compose_banners .upload_banner').remove();` is fine! Part of why I made this list of classnames was to deduplicate code for the warning banners, and that's not really relevant for upload